near-sdk/src/test_utils/test_env.rs

```diff
@@ -1,15 +1,15 @@
 use crate::test_utils::VMContextBuilder;
-use crate::{env, AccountId, MockedBlockchain, VMConfig};
+use crate::{env, MockedBlockchain, VMConfig};
 
-pub fn alice() -> AccountId {
+pub fn alice() -> String {
```

Comment:
I think it would make sense to deprecate these eventually, but was trying to avoid changing the type for this to make migration easier. This is just testing utils but I suppose this should be updated